cobs_test.c

```diff
@@ -2,257 +2,257 @@
+		if ((value) != (expected)) {                                                       \
+			printf("%30s: Failed, %s != %s. Expected 0x%02X, got 0x%02X\n", __func__,  \
+			       #value, #expected, (expected), (value));                            \
+			return false;                                                              \
+		}                                                                                  \
+	} while (0)
+
+#define ASSERT_EQUAL_MEM(value, expected, length)                                                  \
+	do {                                                                                       \
+		for (size_t i = 0; i < (length); i++) {                                            \
+			if ((value)[i] != (expected)[i]) {                                         \
+				printf("%30s: Failed, %s != %s. Expected %s[%lu] = 0x%02X, got "   \
+				       "0x%02X\n",                                                 \
+				       __func__, #value, #expected, #value, i, (expected)[i],      \
+				       (value)[i]);                                                \
+				return false;                                                      \
+			}                                                                          \
+		}                                                                                  \
+	} while (0)
+
+#define ROUNDTRIP_TEST_RUNNER(input, length)                                                       \
+	do {                                                                                       \
+		size_t encoded_buffer_length = (length) + (length) / 254 + 2;                      \
+		uint8_t *encoded_buffer = malloc(encoded_buffer_length);                           \
+		uint8_t *decoded_buffer = malloc(length + 1);                                      \
+		memset(encoded_buffer, 0xAB, encoded_buffer_length);                               \
+		decoded_buffer[length] = 0xAB;                                                     \
+                                                                                                   \
+		size_t encoded_length = cobs_encode(input, length, encoded_buffer);                \
+		ASSERT_LTE_LUINT(encoded_length, length + (length) / 254 + 1);                     \
+		ASSERT_EQUAL_UINT8_HEX(encoded_buffer[encoded_length], 0xAB);                      \
+		ASSERT_EQUAL_UINT8_HEX(encoded_buffer[encoded_buffer_length - 1], 0xAB);           \
+                                                                                                   \
+		size_t decoded_length =                                                            \
+			cobs_decode(encoded_buffer, encoded_length, decoded_buffer);               \
+                                                                                                   \
+		/*      printf("Original:" );                                                      \
```

Comment:
this comment block was not fixed, but it was indented in a way it shouldn't have been

Reply:
what do you mean by that?

Reply:
The code in the comment block that started here https://github.com/GCX-EMBEDDED/cobs/blob/main/cobs_test.c#L61, was aligned with the previous lines. Now it's slightly offset to the right and doesn't align with the other code. I don't get why this alignment change happened, besides the move of the trailing '\'

Reply:
well we are using clang-format now and that tool (with the config we chose) wants to format it that way. So the new style is by definition the correct one.

Reply:
OK, if that's what clang wants we'll keep it like this. For my curiosity, does the comment block look misaligned to you?

Reply:
Not if don't expect different alignment just because there's code in there. The formatter doesn't know it's commented code. That's basically an unspecified concept.